docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  postgres:
+    image: mysql:latest
+    ports: 
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: zintranet
+      MYSQL_ROOT_PASSWORD: root 
```

Comment:
We could use the same env vars as in the app here. Docker Compose automatically reads `.env`. That would ensure the database container and app are always in sync for the db config.

Reply:
I've been wondering about this, clever-cloud addons expose the MYSQL_ADDON_* env variables and the docker-compose file uses MYSQL_* env variables. I'm not sure how to do this then